presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/PartitionTransforms.java

```diff
@@ -253,6 +257,27 @@ private static Block extractTimestamp(Block block, LongUnaryOperator function)
         return builder.build();
     }
 
+    private static Block hoursFromTime(Block block)
+    {
+        return extractHoursFromTime(block, HOURS_DURATION::getValueAsLong);
+    }
+
+    private static Block extractHoursFromTime(Block block, LongUnaryOperator function)
```

Comment:
Name this `extractTime` since it's not specific to hours

Reply:
Changed.